packages/perseus/package.json

```diff
@@ -32,28 +32,27 @@
         "@khanacademy/perseus-linter": "^0.3.12",
         "@khanacademy/pure-markdown": "^0.3.1",
         "@khanacademy/simple-markdown": "^0.11.4",
-        "@khanacademy/wonder-blocks-banner": "^3.0.33",
         "mafs": "https://github.com/Khan/mafs/releases/download/v0.18.1%2Bbuild-cd8b2467893342ba0b02100073fe21f93f8b1125/mafs-0.18.1.tgz"
     },
     "devDependencies": {
-        "@khanacademy/wonder-blocks-button": "^6.2.8",
-        "@khanacademy/wonder-blocks-clickable": "^4.1.0",
-        "@khanacademy/wonder-blocks-core": "^6.3.1",
-        "@khanacademy/wonder-blocks-data": "^13.0.6",
-        "@khanacademy/wonder-blocks-dropdown": "^5.1.6",
-        "@khanacademy/wonder-blocks-form": "^4.4.3",
+        "@khanacademy/wonder-blocks-banner": "^3.0.41",
```

Comment:
**praise:** yay! thanks for upgrading to the latest Banner version (I think the previous one was still using `wb-color` internally).